tests/construction/build_test.py

```diff
@@ -157,6 +157,7 @@ async def test_build_metric_with_dimensions_filters(request):
     )
     expected = """
     SELECT
+        basic_DOT_dimension_DOT_users.age,
```

Comment:
@CircArgs this new line seems incorrect. If there was a group_by on `users.age` defined byt the caller then it would make sense to have it here but then the GROUP_BY clause is missing.

Reply:
Yeah... but if you want that SQL to be valid then any non-agg column you add to SELECT clause you also need to add to GROUP_BY clause.

Reply:
Although... I think that in this case (of filter columns) we don't want to add them to SELECT clause.

Reply:
shoot!... you're right. I'll fix this.